src/main/resources/templates/user-reset.html

```diff
@@ -20,20 +21,18 @@ <h1 class="heading-xlarge heading heading--page-heading">
                         Enter your email
                     </h1>
 
-                    <form method="post" class="push-bottom" th:object="${resetForm}">
-
-                        <div class="form-group" th:classappend="${#fields.hasErrors('email')} ? form-group-error : ''">
-                            <label th:for="email" class="form-label">
-                                Email address
-                            </label>
-                            <span th:if="${#fields.hasErrors('${resetForm.*}')}" class="error-message">
-                                Email address is invalid
-                            </span>
-                            <input class="form-control form-control-2-3" id="email" type="email" th:field="*{email}">
-                        </div>
-
-                        <input type="submit" class="button" value="Continue">
-                        <input type="hidden" th:name="${_csrf.parameterName}" th:value="${_csrf.token}" />
+                    <form action="#" th:action="@{/reset}" method="POST">
+                        <table class="table">
+                            <tr>
+                                <td>
+                                    <label class="d-none" th:for="name">Email:</label>
+                                    <input type="text" name="email"/>
+                                </td>
+                                <td>
+                                    <button type="submit" class="btn btn-primary">Reset</button>
+                                </td>
+                            </tr>
+                        </table>
```

Comment:
Removed error handling for now as we're not using an object within this form

Reply:
ok but where did the csrf go?

Reply:
Spring/Thymeleaf handles it by default